### PR TITLE
Add formulaic package.

### PR DESCRIPTION
**Pull request by matthewwardrop**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example). (inside of source package)
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by matthewwardrop** [comment]
@conda-forge/python Ready for review!

File: recipes/formulaic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/formulaic/meta.yaml:1]: Please remove the guide comments.

File: recipes/formulaic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/formulaic/meta.yaml:27]: Please use the same python constraint from run here, `>=3.6`

File: recipes/formulaic/meta.yaml (hunk 1 above)
Comment (matthewwardrop) [reply recipes/formulaic/meta.yaml]: Done; and apologies.

File: recipes/formulaic/meta.yaml (hunk 2 above)
Comment (matthewwardrop) [reply recipes/formulaic/meta.yaml]: Done.

**Comment by matthewwardrop** [comment]
@conda-forge/python @ocefpaf Ready for review; hopefully for real this time :).

**Pull request merged**